### ISSUE TITLE
Merger doesn't merge files with capital changes

### ISSUE THREAD

**Issue by Marlamin** [wowdev/WoWDBDefs#37]
Such as LfgDungeons which is now LFGDungeons.

**Comment by Marlamin** [comment]
Not exactly a merger issue, but a Windows issue. Merger works fine as is on case sensitive file systems.

Affected files that merge fine on my Ubuntu box:

```
        definitions/BannedAddons.dbd
        definitions/Keychain.dbd
        definitions/LFGDungeonExpansion.dbd
        definitions/LFGDungeonGroup.dbd
        definitions/LFGDungeons.dbd
        definitions/LFGRoleRequirement.dbd
        definitions/PVPDifficulty.dbd
        definitions/PVPItem.dbd
```

Yay.

**Comment by barncastle** [comment]
fixed as per 3f6e6e080919e236464b5ffd75234b325a602a53
